tests/unit/Core/Content/Product/SalesChannel/Detail/ProductDetailRouteTest.php

```diff
@@ -64,6 +70,7 @@ protected function setUp(): void
         $this->idsCollection = new IdsCollection();
         $this->productRepository = $this->createMock(SalesChannelRepository::class);
         $this->systemConfig = $this->createMock(SystemConfigService::class);
+        $this->connection = $this->createMock(Connection::class);
```

Comment:
was this method not tested? as you didn't had to configure the connection mock 🤔

Reply:
No currently there is no test for this method.

Reply:
I have added a new test for this method. I don't know if it works properly in all scenarios, so please check it out. 👍